lca_website/ngo/models.py

```diff
@@ -1,3 +1,27 @@
 from django.db import models
 
-# Create your models here.
+TYPE_ZERO_NGO = 0
+TYPE_ONE_NGO = 1
+
+NGO_CATEGORY = (
+    (TYPE_ZERO_NGO, "Zero"),
+    (TYPE_ONE_NGO, "One"),
+)
+
+
+class ngo(models.Model):
+    """
+    A model to represent an NGO. The Registration number of the NGO is used as the primary key in the database. 
+    Since the category is chosen from a set of categories, it is made into an enum and stored as an integer.
+    The website is made an optional field.
+    The NGO is unapproved when added and has to be approved by the administrators.
+    """
+
+    name = models.CharField(max_length=100)
+    reg_number = models.CharField(max_length=25, primary_key=True)
```

Comment:
Is the registration number a number assigned by government to NGOs or something that we assign to them?

Reply:
I'm assuming it's assigned by the government since it's an input? Since we are taking this input from the NGO